src/sentry/incidents/grouptype.py

```diff
@@ -54,11 +75,10 @@ def extract_dedupe_value(self, data_packet: DataPacket[QuerySubscriptionUpdate])
     def extract_group_values(
         self, data_packet: DataPacket[MetricDetectorUpdate]
     ) -> dict[DetectorGroupKey, int]:
+        # TODO what should the DetectorGroupKey be?
```

Comment:
`None` is correct -- this is the part of code i'm refactoring. we shouldn't need to have grouping rules unless we support grouping in the data packet.

In the future, this would be to pull out the grouped data from the data packet.